packages/graphql-language-service/package.json

```diff
@@ -44,10 +44,10 @@
     "@types/json-schema": "7.0.9",
     "benchmark": "^2.1.4",
     "graphql": "^16.4.0",
-    "graphql-config": "4.3.0",
+    "graphql-config": "4.3.3",
     "lodash": "^4.17.15",
     "platform": "^1.3.5",
-    "ts-node": "^8.10.2",
```

Comment:
wait.... how did we end up with this dependency in the first place? I think it was added by the GSOC intern who added the benchmarks. It's unfortunate that dev dependency resolutions like this impact the whole build just for a benchmark suite!